.gitpod.yml

```diff
@@ -1,6 +1,13 @@
 image: gitpod/workspace-full
 
 tasks:
+  - env:
+      # https://github.com/gitpod-io/gitpod/issues/479
+      PIP_USER: no
+  - init:
+      brew install pre-commit
```

Comment:
This takes a good amount of time on every ws startup. Can it be installed as part of the image?

Reply:
I'll back it out of this pull-request and move 🆙 into workspace-full by default.